src/Backups/BackupIO_S3.cpp

```diff
@@ -187,9 +187,9 @@ void BackupWriterS3::copyFileNative(DiskPtr src_disk, const String & src_file_na
         throw Exception(ErrorCodes::LOGICAL_ERROR, "Cannot natively copy data to disk without source disk");
 
     auto objects = src_disk->getStorageObjects(src_file_name);
-    if (objects.size() > 1)
+    if (objects.size() > 1 || has_throttling)
```

Comment:
It seems it's better not to disable server-side copy is throttling is enabled. Because server-side copy is executed on S3 server and it almost doesn't consume network.

Reply:
Yes, you are right, actually it was my intention, good catch.

Reply:
Fixed.